src/archivematicaCommon/lib/databaseFunctions.py

```diff
@@ -20,84 +20,181 @@
 # @package Archivematica
 # @subpackage archivematicaCommon
 # @author Joseph Perry <joseph@artefactual.com>
+from datetime import datetime
 import os
+import string
 import sys
-import databaseInterface
-import MySQLdb
 import uuid
-from archivematicaFunctions import unicodeToStr
 
-def insertIntoFiles(fileUUID, filePath, enteredSystem=databaseInterface.getUTCDate(), transferUUID="", sipUUID="", use="original"):
+sys.path.append("/usr/share/archivematica/dashboard")
+from main.models import Derivation, Event, File, FileID, FPCommandOutput, Job, SIP, Task, UnitVariable
+
+def getUTCDate():
+    """Returns a string of the UTC date & time in ISO format"""
+    d = datetime.utcnow()
+    return d.isoformat('T')
+
+def getDeciDate(date):
+    valid = "." + string.digits
+    ret = ""
+    for c in date:
+        if c in valid:
+            ret += c
+        #else:
+            #ret += replacementChar
+    return str("{:10.10f}".format(float(ret)))
+
+def insertIntoFiles(fileUUID, filePath, enteredSystem=getUTCDate(), transferUUID="", sipUUID="", use="original"):
+    """
```

Comment:
Yeah, I'm not happy this is a common thing. Mind if I make it a followup PR to make this change? I'm focusing on not changing functionality in this one.